js/src/personal/kazi/components/Hero/Hero.module.css

```diff
@@ -0,0 +1,47 @@
+.wrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 49px;
+  margin-left: 10%;
+  margin-right: 10%;
+  z-index: 1;
```

Comment:
What is the zindex for?

Reply:
z-index is used for the order in which things would stack vertically. Since I want them all to be on the same level, putting z-index of 1 helps me with that.